back/engines/commercial/content_builder/app/models/content_builder/layout_image.rb

```diff
@@ -14,14 +14,12 @@ module ContentBuilder
   class LayoutImage < ApplicationRecord
     mount_base64_uploader :image, LayoutImageUploader
 
-    validates :image, presence: true
```

Comment:
Although removing this seems wrong, I suspect it is 'correct'.

I note that both `project_image` and `idea_image`, which are the models most similar to `image_layout`, also do not validate the presence of `image`.

It appears necessary to remove this validation to prevent the error I encountered when applying a template containing `layout_images` with this validation in place:
```Ruby
"model_class":"ContentBuilder::LayoutImage","attributes":{"code":"1d20c7fd-ba04-4e15-a2fa-84869dce19d0","remote_image_url":"http://localhost:4000/uploads/affad46f-46b1-4824-b845-5926432042e5/content_builder/layout_image/image/aff20487-cbf8-49a4-b81c-c709807e1e76/6f4b9583-8464-4bca-8f7a-76e7ae564214.jpeg","created_at":"2023-02-04 12:35:09 UTC","updated_at":"2023-02-04 12:35:09 UTC"}}
from /cl2_back/engines/commercial/multi_tenancy/app/services/multi_tenancy/tenant_template_service.rb:111:in `rescue in block (2 levels) in apply_template'
Caused by ActiveRecord::RecordInvalid: Validation failed: Image can't be blank
from /bundle/ruby/2.7.0/gems/activerecord-6.1.7.1/lib/active_record/validations.rb:80:in `raise_validation_error'
```

The absence of these validations form such image files, however, could well be part of the reason we sometimes find multiple image records, with null values for `image`. I have noted this on the ticket related to the discovery of multiple idea_images with such null values ([CL-2868](https://citizenlab.atlassian.net/browse/CL-2686)).